<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>440</width>
    <height>328</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>单词处理</string>
  </property>
  <property name="styleSheet">
   <string notr="true">border:1px groove gray;border-radius:10px;</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>20</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft YaHei</family>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border:0px </string>
   </property>
   <property name="text">
    <string>比对文件地址：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="file_adress_bidui">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>20</y>
     <width>211</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="select_file_bidui">
   <property name="geometry">
    <rect>
     <x>344</x>
     <y>20</y>
     <width>61</width>
     <height>23</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft YaHei</family>
    </font>
   </property>
   <property name="text">
    <string>选择文件</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>50</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft YaHei</family>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border:0px</string>
   </property>
   <property name="text">
    <string>考纲词文件地址：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="file_adress_kaogang">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>50</y>
     <width>211</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="select_file_kaogang">
   <property name="geometry">
    <rect>
     <x>344</x>
     <y>50</y>
     <width>61</width>
     <height>23</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft YaHei</family>
    </font>
   </property>
   <property name="text">
    <string>选择文件</string>
   </property>
  </widget>
  <widget class="QPushButton" name="save">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>280</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft YaHei</family>
    </font>
   </property>
   <property name="text">
    <string>保存</string>
   </property>
  </widget>
  <widget class="QComboBox" name="select_type">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>280</y>
     <width>191</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft YaHei</family>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border:2px groove gray;border-radius:10px;padding:2px 4px;</string>
   </property>
   <item>
    <property name="text">
     <string>选择要保存的单词</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>保存对比文件中的所有单词</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>保存对比文件中的考纲词</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>保存对比文件中缺失的考纲词</string>
    </property>
   </item>
  </widget>
  <widget class="QPushButton" name="jiexi">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>280</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft YaHei</family>
    </font>
   </property>
   <property name="text">
    <string>解析</string>
   </property>
  </widget>
  <widget class="QComboBox" name="select_file_type">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>280</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft YaHei</family>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border:2px groove gray;border-radius:10px;padding:2px 4px;</string>
   </property>
   <item>
    <property name="text">
     <string>保存格式</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>.csv</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>.txt</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>.xlsx</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>80</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft YaHei</family>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border:0px</string>
   </property>
   <property name="text">
    <string>文件保存地址：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="file_save_path">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>80</y>
     <width>211</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="select_save_path">
   <property name="geometry">
    <rect>
     <x>344</x>
     <y>80</y>
     <width>61</width>
     <height>23</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft YaHei</family>
    </font>
   </property>
   <property name="text">
    <string>选择路径</string>
   </property>
  </widget>
  <widget class="QScrollArea" name="scrollArea">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>110</y>
     <width>381</width>
     <height>161</height>
    </rect>
   </property>
   <property name="verticalScrollBarPolicy">
    <enum>Qt::ScrollBarAsNeeded</enum>
   </property>
   <property name="horizontalScrollBarPolicy">
    <enum>Qt::ScrollBarAsNeeded</enum>
   </property>
   <property name="sizeAdjustPolicy">
    <enum>QAbstractScrollArea::AdjustIgnored</enum>
   </property>
   <property name="widgetResizable">
    <bool>true</bool>
   </property>
   <widget class="QWidget" name="scrollAreaWidgetContents">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>379</width>
      <height>159</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="msg">
       <property name="font">
        <font>
         <family>Microsoft YaHei</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">border:0px
</string>
       </property>
       <property name="text">
        <string>欢迎使用！</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
